bin/download-local-dev-data.sh

```diff
@@ -0,0 +1,35 @@
+# "normally" the app downloads the latest data from Overpass API based on the
+# user's clipped areas.
+#
+# For expediency and deterministic tests, this script downloads some
+# pre-configured areas, which the web app will load from localhost.
+
+cd "$(git rev-parse --show-toplevel)"
+
+download_to_subdir() {
+    local subdir=$1
+    local url=$2
+
+    mkdir -p "$subdir"
+    (wget -P "$subdir" --timestamping "$url" && echo "✅ $url") \
+        || echo "❌ Download failed: $url"
+}
+
+cd web/public
+
+download_to_subdir osm https://assets.od2net.org/severance_pbfs/areas.json
+
+# Global data used for tests and demo data
+AREAS="bristol edinburgh strasbourg ut_dallas"
+for x in $AREAS; do
+    download_to_subdir osm "https://assets.od2net.org/severance_pbfs/$x.pbf"
+    download_to_subdir boundaries "https://assets.od2net.org/boundaries/$x.geojson"
+done
+
+# Scotland specific data
+jq '.features[] | .properties.kind + "_" + .properties.name' ../../data_prep/scotland/boundaries.geojson | sed 's/"//g' | while read x; do
+    download_to_subdir cnt_boundaries "https://assets.od2net.org/cnt_boundaries/$x.geojson"
+    download_to_subdir cnt_osm "https://assets.od2net.org/cnt_osm/$x.osm.pbf"
+    # OD Demand model
+    download_to_subdir cnt_demand "https://assets.od2net.org/cnt_demand/$x.bin"
```

Comment:
Oops, the original script in the readme was wrong. The files have `demand_` prefixed. I was originally thinking of not putting them in their own directory. Could rename later to remove redundancy.

```suggestion
    download_to_subdir cnt_demand "https://assets.od2net.org/cnt_demand/demand_$x.bin"
```